inc/core/templates/theme-notification-banner.php

```diff
@@ -0,0 +1,315 @@
+												}
+                                                ?>
+												/>
+										<span class="message">Make the text bold</span> </label>
+
+									<label>
+										<input 	type="checkbox" 
+												id="notice_label_big" 				
+												name="notice_label_big" 
+												value="1" 
+												<?php
+                                                if ( $notice_label_big ) {
+													echo ' checked="checked"';
+												}
+                                                ?>
+												/>
+										<span class="message">Make the text large</span> </label>
+								</div>
+							</div>
+						</div>
+						<div class="field full">
+							<div class="label">
+								<label for="notice_content">Notification content</label>
+							</div>
+							<div class="input">
+								<div class="input-wrap"><input type="text" id="notice_content" name="notice_content" value="<?php if ( ! empty( $notice_content ) ) { echo esc_attr( $notice_content ); } ?>"></div>
+								<p class="description">Please enter the notice information in plain text. HTML or rich text markup is not allowed. To add a link, select the 'enable button' checkbox below.</p>
+							</div>
+						</div>
+						<div class="field">
+							<div class="label">
+								<label for="notice_button_enabled">Enable button</label>
+							</div>
+							<div class="input">
+								<div class="true-false">
+									<label>
+										<input 	type="checkbox" 
+												id="notice_button_enabled" 				
+												name="notice_button_enabled" 
+												value="1" 
+												<?php
+                                                if ( $notice_button_enabled ) {
+													echo ' checked="checked"';
+												}
+                                                ?>
+												/>
+										<span class="message">Add a button / link</span> </label>
+								</div>
+							</div>
+						</div>
+						<div class="field">
+							<div class="label">
+								<label for="notice_button_label">Button Label</label>
+							</div>
+							<div class="input">
+								<div class="input-wrap"><input type="text" id="notice_button_label" name="notice_button_label"
+								required
+								value="<?php if ( ! empty( $notice_button_label ) ) { echo esc_attr( $notice_button_label ); } ?>" maxlength="32"></div>
+							</div>
+						</div>
+						<div class="field half">
+							<div class="label">
+								<label for="notice_button_aria_label">Accessibility button label</label>
+							</div>
+							<div class="input">
+								<div class="input-wrap"><input type="text" id="notice_button_aria_label" name="notice_button_aria_label" value="<?php if ( ! empty( $notice_button_aria_label ) ) { echo esc_attr( $notice_button_aria_label ); } ?>"></div>
+								<p class="description">If the button label is not informative, add a more descriptive label designed to help assistive technology (e.g. screen readers). This will not be visible.</p>
```

Comment:
I appreciate you describing the purpose of this to the user as many won't know what an aria label is.